specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/bastionHost.json

```diff
@@ -290,6 +262,255 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/bastionHosts/{bastionHostName}/createbsl": {
+      "post": {
+        "tags": [
+          "BastionHosts",
+          "BastionShareableLink"
+        ],
+        "operationId": "PutBastionShareableLink",
+        "description": "Creates a Bastion Shareable Links for all the VMs specified in the request.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/BastionHostName"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/BastionShareableLinkRequest"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of shareable links for the specified VMs, giving appropriate error messages as needed.",
+            "schema": {
+              "$ref": "#/definitions/BastionShareableLinkListResult"
+            }
+          },
+          "202": {
```

Comment:
Async APIs should include:

        "x-ms-long-running-operation": true,
        "x-ms-long-running-operation-options": {
          "final-state-via": "___________"
        }